drivers/starburst/src/metabase/driver/implementation/execute.clj

```diff
@@ -70,6 +71,14 @@
           (throw e)))
         (f conn))))
 
+(defmethod sql-jdbc.conn/data-warehouse-connection-pool-properties :starburst
+  [driver database]
+  ;; In order to avoid Metabase from cancelling long queries, the
+  ;; unreturnedConnectionTimeout property needs to be updated
+  (merge
+   ((get-method sql-jdbc.conn/data-warehouse-connection-pool-properties :sql-jdbc) driver database)
+   {"unreturnedConnectionTimeout" 100000000000}))
```

Comment:
How long is this?

Reply:
The timeout is in seconds, so it should be enough to avoid any timeout of this type

Reply:
Is this a default value? Do we really want no timeout?